src/sql-lexer/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "mz-sql-lexer"
+description = "The lexer for Materialize's SQL dialect."
+version = "0.0.0"
+edition.workspace = true
+rust-version.workspace = true
+publish = false
+
+[dependencies]
+mz-ore = { path = "../ore", default-features = false, features = ["stack"] }
+workspace-hack = { version = "0.0.0", path = "../workspace-hack", optional = true }
+serde = { version = "1.0.152", features = ["derive"] }
+phf = { version = "0.11.1", features = ["uncased"] }
+uncased = "0.9.7"
```

Comment:
I don't see these being using in lexer.rs Is it sufficient to have them only on build-deps?

Reply:
`uncased is by `keywords.rs`, and `phf` is pulled in with the `include!`